Makefile

```diff
@@ -56,8 +61,28 @@ clean-docker-all: clean-docker-containers clean-docker-buildcache clean-docker-i
 
 clean-all: clean-docker-buildcache clean-docker-containers
 
-test:
-	make start-lumigator-build
-	cd lumigator/python/mzai/backend; SQLALCHEMY_DATABASE_URL=sqlite:///local.db uv run pytest
-	cd lumigator/python/mzai/sdk; uv run pytest -o python_files="test_*.py int_test_*.py"
-	make stop-lumigator
+sdk-unit-test:
+	cd lumigator/python/mzai/sdk/tests; \
+	uv run pytest -o python_files="unit/*/test_*.py unit/test_*.py"
+
+sdk-int-test-exec:
+	cd lumigator/python/mzai/sdk/tests; \
+	uv run pytest -o python_files="int/test_*.py int/*/test_*.py"
+
+sdk-int-test: | start-lumigator-build sdk-int-test-exec stop-lumigator
+
+sdk-test: sdk-unit-test sdk-int-test
+
+backend-unit-test:
+	cd lumigator/python/mzai/backend; \
```

Comment:
We don't currently do `; \` in the rest of the makefile, so let's either remove this or add it to all the other commands for consistency

Reply:
even for multi-command targets? I didn't check, let me give it a look.